goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["gun-part",                                   "gun-part",                   3, ["ENGINE", "GAME"], "engine/target/gun"],
+["collide-reaction-target",                    "collide-reaction-target",    3, ["ENGINE", "GAME"], "engine/collide"],
+["logic-target",                               "logic-target",               3, ["ENGINE", "GAME"], "engine/target"],
+["sidekick",                                   "sidekick",                   3, ["ENGINE", "GAME"], "engine/target"],
+["voicebox",                                   "voicebox",                   3, ["ENGINE", "GAME"], "engine/game"],
+["collectables-part",                          "collectables-part",          3, ["ENGINE", "GAME"], "engine/game"],
+["debug-part",                                 "debug-part",                 3, ["ENGINE", "GAME"], "engine/debug"],
+["find-nearest",                               "find-nearest",               3, ["ENGINE", "GAME"], "engine/util"],
+["task-arrow",                                 "task-arrow",                 3, ["ENGINE", "GAME"], "engine/game/task"],
+["projectile",                                 "projectile",                 3, ["ENGINE", "GAME"], "engine/game"],
+["target-handler",                             "target-handler",             3, ["ENGINE", "GAME"], "engine/target"],
+["target-anim",                                "target-anim",                3, ["ENGINE", "GAME"], "engine/target"],
+["target",                                     "target",                     3, ["ENGINE", "GAME"], "engine/target"],
+["target2",                                    "target2",                    3, ["ENGINE", "GAME"], "engine/target"],
+["target-swim",                                "target-swim",                3, ["ENGINE", "GAME"], "engine/target"],
+["target-carry",                               "target-carry",               3, ["ENGINE", "GAME"], "engine/target"],
+["target-darkjak",                             "target-darkjak",             3, ["ENGINE", "GAME"], "engine/target"],
+["target-death",                               "target-death",               3, ["ENGINE", "GAME"], "engine/target"],
+["target-gun",                                 "target-gun",                 3, ["ENGINE", "GAME"], "engine/target"],
+["gun-util",                                   "gun-util",                   3, ["ENGINE", "GAME"], "engine/target/gun"],
+["gun-blue-shot",                              "gun-blue-shot",              3, ["ENGINE", "GAME"], "engine/target/gun"],
+["gun-yellow-shot",                            "gun-yellow-shot",            3, ["ENGINE", "GAME"], "engine/target/gun"],
+["gun-red-shot",                               "gun-red-shot",               3, ["ENGINE", "GAME"], "engine/target/gun"],
+["gun-dark-shot",                              "gun-dark-shot",              3, ["ENGINE", "GAME"], "engine/target/gun"],
+["gun-states",                                 "gun-states",                 3, ["ENGINE", "GAME"], "engine/game"],
```

Comment:
`target/gun` - this is stuff like `target-gun-walk`, `target-gun-post` that are add-ons to the normal target post